tests/check-registry.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+registry=$1
+
+if grep -q "$registry:_authToken=\${NODE_AUTH_TOKEN}" ./.npmrc; then
```

Comment:
let's use `npm config get registry` instead of reading the file; the goal here is to ensure that things _actually_ work (vs just checking the side effects that we wrote ourselves)